From: Michael Biebl <biebl@debian.org>
Date: Tue, 2 Jan 2024 11:09:56 +0100
Subject: test: deny-list TEST-13-NSPAWN

The default ram size of 1024M for qemu virt is not sufficient
to make the test pass reliably on Debian sid/trixie.
Disable the test for now until this has been addressed in debci:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1059840
---
 test/TEST-13-NSPAWN/deny-list-upstream-ci | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 test/TEST-13-NSPAWN/deny-list-upstream-ci

diff --git a/test/TEST-13-NSPAWN/deny-list-upstream-ci b/test/TEST-13-NSPAWN/deny-list-upstream-ci
new file mode 100644
index 0000000..2a8a53a
--- /dev/null
+++ b/test/TEST-13-NSPAWN/deny-list-upstream-ci
@@ -0,0 +1,4 @@
+# The default ram size of 1024M for qemu virt is not sufficient
+# to make the test pass reliably on Debian sid/trixie.
+# Disable the test for now until this has been addressed in debci:
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1059840
